lib/groonga-query-log/response-comparer.rb

```diff
@@ -59,16 +59,61 @@ def same_response?
       @response1.body == @response2.body
     end
 
+    def same_sort_key?
+      return false unless @command.has_key?(:sort_keys)
+      if !@command.sort_keys.empty?
+        @response1.records.each do |record1|
+          @response2.records.each do |record2|
+            if record2[@command.sort_keys] == record1[@command.sort_keys]
+              next
+            else
+              return false
+            end
+          end
+        end
+        return true
+      else
+        return false
+      end
+    end
+
+    def same_shard_key?
+      return false unless @command.has_key?(:shard_key)
+      if !@command.shard_key.empty?
+        @response1.records.each do |record1|
```

Comment:
`return false if @command.shard_key.empty? ` 
It seems better to return in early stage.(it also reduces nested block)